src/huggingface_hub/_login.py

```diff
@@ -222,11 +221,11 @@ def auth_list() -> None:
         print(f"{is_current} {{:<{max_offset}}}| {{:<15}}".format(token_name, masked_token))
 
     if _get_token_from_environment():
-        print(
+        logger.info(
```

Comment:
```suggestion
        logger.warning(
```

(nit) maybe a warning here as it might lead to an unexpected behavior for the user (if not aware of it)